docs/get_started.rst

```diff
@@ -1,17 +1,348 @@
-Get started
+Get Started
 ===========
 
+.. contents::
+    :depth: 2
+    :local:
+
+
 Install CumulusCI
 -----------------
 
+.. note:: These installation instructions assume some familiarity with entering commands into a terminal.
+    If that's completely new to you, we recommend starting with the `CumulusCI setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ module on Trailhead to walk you through step by step.
+
+
+On macOS
+^^^^^^^^
+Ensure you have `Homebrew <https://brew.sh/>`_ installed as it is a prerequisite for installing CumulusCI on macOS.
+
+
+Install via ``pipx``
+******************************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+First install ``pipx`` with the following two commands::
+
+    $ brew install pipx
+    $ pipx ensurepath
+
+
+Once ``pipx`` is installed, you can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Linux
+^^^^^^^^
+
+
+
+Install via ``pipx``
+**************************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+Installation instruction for ``pipx`` can be found `here <https://pipxproject.github.io/pipx/installation/>`_.
+Once ``pipx`` is installed, we can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Windows
+^^^^^^^^^^
+
+
+
+Install Python 3
+********************
+#. Go to the `Python downloads page <https://www.python.org/downloads/release/python-383/>`_.
+#. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
+#. Use the installation wizard to install:
+
+   * Check the "Add Python <version> to PATH" checkbox
+   * Click "Install Now"
+
+.. image:: images/windows_python.png
+
+
+
+Install via ``pipx``
+***********************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+Open your preferred terminal application
+(e.g. `CMD.exe <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd>`_ on Windows).
+If you already have your terminal open, close it and reopen it. Enter the following command::
+
+    $ python -m pip install --user pipx
+
+.. image:: images/pipx.png
+
+Next we want to modify the default PATH environment variable to include pipx::
+
+    $ python -m pipx ensurepath
```

Comment:
Unfortunately pipx ensurepath doesn't seem to do anything on my son's computer. :( 

I can investigate further, but just a warning that this isn't bulletproof.

WINDOWS!!!